.github/workflows/linux-snap.yml

```diff
@@ -0,0 +1,43 @@
+name: Linux Snap
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  Snap:
+    runs-on: ubuntu-latest
+
+    env:
+      SNAPCRAFT_BUILD_INFO: 1
+
+    timeout-minutes: 60
+    steps:
+    - name: Install Snapcraft
+      uses: samuelmeuli/action-snapcraft@v1
+      with:
+        snapcraft_token: ${{ secrets.SNAPCRAFT_TOKEN }}
+        use_lxd: true
+
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        # Needed for version determination to work
+        fetch-depth: 0
+
+    - name: Build
+      run: |
+        # Build the `subsurface` part.
+        sg lxd -c '/snap/bin/snapcraft build --use-lxd subsurface'
+
+    - name: Build and verify the snap
+      id: build-snap
+      env:
+        SNAP_ENFORCE_RESQUASHFS: 0
+      run: |
+        # Actually build the snap.
+        sg lxd -c '/snap/bin/snapcraft --use-lxd'
```

Comment:
that looks... odd... but I'll take your word for it :-)

Reply:
the point was that in the previous invocation of basically the same command line, there's a reference to `subsurface`, vs here it's just `/snap/bin/snapcraft --use-lxd` without further argument.

Reply:
Ah yes, this is implicitly snapcraft snap, kinda like make makes the default target.